cook-book/getting-started/hosts.yml

```diff
@@ -0,0 +1,55 @@
+---
+all:
+  vars:
+    cartridge_app_name: myapp
+    cartridge_cluster_cookie: secret-cookie
+
+    cartridge_package_path: ./myapp-1.0.0-0.rpm
+
+    cartridge_bootstrap_vshard: false
+
+    # may be useful for vagrant
+    ansible_ssh_private_key_file: ~/.vagrant.d/insecure_private_key
+    ansible_ssh_common_args: '-o IdentitiesOnly=yes -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no'
+
+  hosts:  # instances
+    storage-1:
+      config:
+        advertise_uri: '172.19.0.3:3303'
+        http_port: 8183
+
+    storage-1-replica:
+      config:
+        advertise_uri: '172.19.0.2:3302'
+        http_port: 8182
+
+  children:
+    # group instances by machines
+    machine_1:
+      vars:
+        ansible_host: 172.19.0.2
+        ansible_user: vagrant
```

Comment:
What about to use ansible_user in global vars?

Reply:
I think that it's OK to place connection opts together.